geemusic/controllers.py

```diff
@@ -17,5 +18,5 @@ def redirect_to_stream(song_id):
         )
 
     app.logger.debug('URL is %s' % stream_url)
-
-    return redirect(stream_url, code=302)
+    req = requests.get(stream_url, stream=False)
+    return Response(stream_with_context(req.iter_content(chunk_size=1024 * 1024)), content_type=req.headers['content-type'])
```

Comment:
Copy pasting from the issue.

>Here's the code. Basically it streams the url in 1mb chunks though the `APP_URL` set in you `.env` instead of redirecting to the google url. This appears to amazon as if you were streaming it from your home server url and not googles. Amazon is blocking google's ssl certificate for `googleusercontent.com` but Amazon accepts your `APP_URL` certificate. Therefore it works (but it needs some fine tuning on the chunk size for slow internet connections).